addons/sprite-layering/userscript.js

```diff
@@ -0,0 +1,142 @@
+export default async function ({ addon, global, console, msg }) {
+  const vm = addon.tab.traps.vm;
+
+  const manager = document.createElement("div");
+  manager.id = "sa-layer-manager";
+  manager.classList.add(addon.tab.scratchClass("asset-panel_wrapper"), "sa-layer-manager");
+
+  const layerTab = document.createElement("li");
+  addon.tab.displayNoneWhileDisabled(layerTab, { display: "flex" });
+  layerTab.classList.add(addon.tab.scratchClass("react-tabs_react-tabs__tab"), addon.tab.scratchClass("gui_tab"));
+  // Cannot use number due to conflict after leaving and re-entering editor
+  layerTab.id = "react-tabs-sa-layer-manager";
+
+  const layerTabIcon = document.createElement("img");
+  layerTabIcon.draggable = false;
+  layerTabIcon.style.marginRight = "0.5rem";
+  layerTabIcon.style.width = "1em";
+  layerTabIcon.src = addon.self.dir + "/icon.svg";
+
+  const layerTabText = document.createElement("span");
+  layerTabText.innerText = "Layers";
+
+  layerTab.appendChild(layerTabIcon);
+  layerTab.appendChild(layerTabText);
+  
+  layerTab.addEventListener("click", (e) => {
+    addon.tab.redux.dispatch({ type: "scratch-gui/navigation/ACTIVATE_TAB", activeTabIndex: 4 });
+  });
+
+  function setVisible(visible) {
+    if (visible) {
+      layerTab.classList.add(
+        addon.tab.scratchClass("react-tabs_react-tabs__tab--selected"),
+        addon.tab.scratchClass("gui_is-selected")
+      );
+      const contentArea = document.querySelector("[class^=gui_tabs]");
+      contentArea.insertAdjacentElement("beforeend", manager);
+      const layerBody = document.getElementById('sa-layer-manager');
+
+      while (layerBody.firstChild) {
+        layerBody.removeChild(layerBody.firstChild);
+      }
+
+      var targets = vm.runtime.targets;
+
+      var sortedTargets = [];
+      var layers = [];
+      for (var i=0;i<targets.length;i++) {
+        layers.push(targets[i].getLayerOrder());
+      }
+
+      var x = 0;
+      for (var i=0;i<layers.length;i++) {
+        x = layers.indexOf(i);
+        sortedTargets.push(targets[x])
+      }
+      
+      for (var z=sortedTargets.length;z>0;z--) {
+        var i = z-1;
+        var layer = document.createElement('div');
+        layer.className = "layer";
+        layer.id = "layer-manager-" + sortedTargets[i].getLayerOrder();
+
+        var layerNum = document.createElement('p');
+        layerNum.className = "layer-id";
+        layerNum.innerHTML = sortedTargets[i].getLayerOrder();
+
+        var spriteName = document.createElement('p');
+        spriteName.className = "sprite-name";
+        if (!sortedTargets[i].isOriginal) {
+          spriteName.innerHTML = sortedTargets[i].getName() + " (clone)";
```

Comment:
Please, use `msg()` [function](https://scratchaddons.com/docs/localization/localizing-addons/) (with placeholders), so we can translate this text.

Reply:
How would I fix the prettier error?